README.md

```diff
@@ -46,10 +46,10 @@ On first run you will be asked for some information so that the app can build yo
 
 You'll need to enter a valid Key Vault name.
 
-Alternatively you can manually pre-load a config file at `~/.azure-keyvault-browser.toml` and use the following schema:
+Alternatively you can manually pre-load a config file at `~/azure-keyvault-browser/config.toml` and use the following schema:
```

Comment:
The config lives in `.config` now

https://github.com/chelnak/azure-keyvault-browser/blob/81f914ee2c2eeb641350610a37af97d941b374eb/src/azure_keyvault_browser/config.py#L23

So i think this should be `~/.config/azure-keyvault-browser/config.toml`

Reply:
Ah yes, of course. Good spot. Fixed in 24270ea6795ee14e6cc1039d30e64e3db18cd4ab.